packages/amazonq/.changes/next-release/Bug Fix-e2bb3c83-52e8-4ae9-aedb-844e8780971b.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Fix the first Amazon Q inline suggestion example"
```

```suggestion
	"description": "Inline: Typos in the first example suggestion"
```
Please follow the changelog guidelines: https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog